### PR TITLE
login page(ui)

### PR DESCRIPTION
**Pull request by helana3**

### CODE REVIEW

File: lib/core/global/custom_text_filed/custom_text_field.dart [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        ) : widget.suffixIcon,
+        fillColor: Colors.white,
+        filled: true,
+        hintText: widget.label,
+        border: OutlineInputBorder(
+          borderRadius: BorderRadius.circular(16),
+        ),
+      ),
+      validator: (value) {
+        if (value == null || value.isEmpty) {
```
Comment (MahmoudElgaml) [inline lib/core/global/custom_text_filed/custom_text_field.dart:52]: there is validatior functions in    core/service/validationService use it

File: lib/core/utils/app_style.dart [hunk 2]
```diff
@@ -11,4 +11,38 @@ class AppStyle {
   );
 
 
+
```
Comment (MahmoudElgaml) [inline lib/core/utils/app_style.dart:14]: use screen utils( 24.sp)

**Review by MahmoudElgaml — CHANGES_REQUESTED** [review]
finish what i hint on code

File: lib/core/utils/app_style.dart (hunk 2 above)
Comment (helana3) [reply lib/core/utils/app_style.dart:14]: m4 fahmak

File: lib/core/global/custom_text_filed/custom_text_field.dart (hunk 1 above)
Comment (helana3) [reply lib/core/global/custom_text_filed/custom_text_field.dart:52]: Done

**Pull request closed without merging**